app/Console/Command/FindMatchingTrailersOnYouTube.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Json;
+use CultuurNet\UDB3\Kinepolis\Trailer\TrailerRepository;
+use CultuurNet\UDB3\Model\ValueObject\Identity\Uuid;
+use CultuurNet\UDB3\Offer\Commands\Video\AddVideo;
+use CultuurNet\UDB3\ReadModel\DocumentDoesNotExist;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\ReadModel\JsonDocument;
+use CultuurNet\UDB3\Search\ResultsGenerator;
+use CultuurNet\UDB3\Search\ResultsGeneratorInterface;
+use CultuurNet\UDB3\Search\SearchServiceInterface;
+use CultuurNet\UDB3\Search\Sorting;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class FindMatchingTrailersOnYouTube extends Command
+{
+    private CommandBus $commandBus;
+
+    private ResultsGeneratorInterface $searchResultsGenerator;
+
+    private DocumentRepository $documentRepository;
+
+    private TrailerRepository $trailerRepository;
+
+    public function __construct(
+        CommandBus $commandBus,
+        SearchServiceInterface $searchService,
+        DocumentRepository $documentRepository,
+        TrailerRepository $trailerRepository
+    ) {
+        parent::__construct();
+        $this->commandBus = $commandBus;
+        $this->searchResultsGenerator = new ResultsGenerator(
+            $searchService,
+            new Sorting('created', 'desc'),
+            100
```

Comment:
I would search more and catch the exception from the quota limit and then just stop?

Reply:
I'm not in favor to just catch the Exception. If we know the limit before, I think it's best to keep it in mind.

Reply:
Also fine, but maybe make it an appconfig then?